### ISSUE TITLE
[Vue] [Dialog]: "update:open" emit not declared

### ISSUE THREAD

**Issue by jiblett1000** [chakra-ui/ark#1035]
### Description

I'm getting this error when closing a dialog component.

`[Vue warn]: Component emitted event "update:open" but it is neither declared in the emits option nor as an "onUpdate:open" prop.`

Everything seems to work just fine, but would be nice to not have this warning in the console. I also tried adding `update:open` to the emits in my custom `ChildDialog` component but that did not help, which leads me to believe it might be something within the Ark `Dialog` causing this.

### Link to Reproduction

https://stackblitz.com/edit/nuxt-starter-cy3vjz?file=package.json,components%2FChildDialog.vue

### Steps to reproduce

1. Click "Open in New Tab".
2. Enable developer console.
3. Open the dialog.
4. Click X to close dialog.
5. Click cancel in confirm dialog.
6. See error in console.

### Ark UI Version

0.4

### Framework

- [ ] React
- [ ] Solid
- [X] Vue

**Comment by jiblett1000** [comment]
is this merged as of 0.5.2? Still getting this

**Comment by arnoson** [comment]
I get something similar in `0.5.3`:

```
[Vue warn]: Component emitted event "open" but it is neither declared in the emits option nor as an "onOpen" prop. 
```
